### DISCUSSION TITLE
Sure, I will help with that:

### DISCUSSION THREAD

**Discussion by EtahReign** [LostRuins/koboldcpp#1190] [Q&A] [answered]
Hello, 

Does anyone know how to prevent "
Sure, I will help with that:" from popping up with ever text output?

**Comment by jabberjabberjabber** [comment]
Settings-> format-> uncheck 'assistant jailbreak'.

**Reply by EtahReign** [reply]
Thank you for the response. That option is already unselected. It still responds with the same prepended verbiage.

**Reply by LostRuins** [reply, marked as answer]
That shouldn't happen. Make sure you press Ok after toggling the jailbreak option off. 

If it still does not work, try pressing "back" twice to undo the most recent 2 actions, then generate again

**Reply by EtahReign** [reply]
Thanks, going back and hitting ok worked. Assistant jailbreak was unchecked and persisted unchecked after reboot previously,  after following your instructions it resolved the issue.
